src/reducers/MapReducer.js

```diff
@@ -58,7 +58,9 @@ export default (state = INITIAL_STATE, action) => {
     switch (type) {
 
         case INITIALIZE_MAP:
-            return payload ? {...INITIAL_STATE, lastKnownRegion: payload} : INITIAL_STATE;
+            const {region} = payload;
+
+            return payload ? {...INITIAL_STATE, lastKnownRegion: region} : INITIAL_STATE;
```

Comment:
Refactored the region to be passed as an object, more in line with how we pass around other things in the app lately.